adapters-2.0/src/test/java/com/flipkart/fdp/ml/adapter/DecisionTreeRegressionModelBridgeTest.java

```diff
@@ -0,0 +1,60 @@
+package com.flipkart.fdp.ml.adapter;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.spark.ml.linalg.SparseVector;
+import org.apache.spark.ml.regression.DecisionTreeRegressionModel;
+import org.apache.spark.ml.regression.DecisionTreeRegressor;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.junit.Test;
+
+import com.flipkart.fdp.ml.export.ModelExporter;
+import com.flipkart.fdp.ml.importer.ModelImporter;
+import com.flipkart.fdp.ml.transformer.DecisionTreeTransformer;
+
+/**
+ * Created by akshay.us on 8/29/16.
+ */
+public class DecisionTreeRegressionModelBridgeTest extends SparkTestBase {
+
```

Comment:
Please add a test using this in a pipeline as well. You can refer 1.6 test for the same.